selfdrive/controls/lib/latcontrol_pid.py

```diff
@@ -2,18 +2,20 @@
 
 from selfdrive.controls.lib.pid import PIController
 from selfdrive.controls.lib.drive_helpers import get_steer_max
+from selfdrive.controls.lib.latcontrol import LatControl, MIN_STEER_SPEED
 from cereal import log
 
 
-class LatControlPID():
+class LatControlPID(LatControl):
   def __init__(self, CP, CI):
+    super().__init__(CP)
```

Comment:
We can also take in `CI` for the other two controllers in controlsd, so you don't have the mismatching arguments